### DISCUSSION TITLE
Can't require js file outside of project dir

### DISCUSSION THREAD

**Discussion by jakobrosenberg** [babel/babel#13693] [Q&A] [answered]
When requiring a `config.js` file outside the project dir, I get the following error.

```
Error: Couldn't find preset "env" relative to directory "/path/to/my-configs"
```

I've tried adding `my-configs` and `/path/to/my-configs` to the ignore option, but to no avail.

**Comment by jakobrosenberg** [comment, marked as answer]
Removing ` --presets=env` from the script, seems to have fixed it.

Is `--presets=env` needed, if the environment is `development` and `.babelrc` contains
```json
  "env": {
    "development": {
      "presets": [["env"]]
    }
  }
```

**Reply by nicolo-ribaudo** [reply]
It's not needed because you are already specifying it in your config. However, if you want to also compile files outside of your project dir, you probably need to update to Babel 7 and use `babel.config.json` rather than `.babelrc` (I don't exactly remember how Babel 6 works).

https://babeljs.io/docs/en/v7-migration

**Reply by jakobrosenberg** [reply]
Thanks for your reply @nicolo-ribaudo. Babel 7 would be great, but red tape and all that... 🙂
